Schemas.py

```diff
@@ -0,0 +1,14 @@
+from database import db
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
What if we name this UserModel.py so that the the folder/package containing the file should be named Schemas.py .
This is just a opinion .
i am totally fine with your approach too ...

Reply:
> our approach too ...

Then it will limit this file to include only user schemas. we have to include other schemas such as notes and pages etc. 
if you want to rename this, then we can add a folder and add these files in that folder.

Reply:
That's what i said boss. that we should name it like usermodel or something similar ...